Documentation/gettingstarted/external-workloads.rst

```diff
@@ -0,0 +1,353 @@
+type that, depending on your cloud provider, allows use of a static
+IP, making configuring the external workloads easier.
+
+.. tabs::
+  .. group-tab:: NodePort
+
+    NodePort is the default service type. Get the Node IP to use with:
+
+    .. code:: bash
+
+      kubectl get node -o jsonpath='{.items[0].status.addresses[?(@.type == "InternalIP")].address}{"\n"}'
+
+  .. group-tab:: GCP
+
+    Add the following values to the ``helm install cilium`` command:
+
+    .. parsed-literal::
+
+      --set clustermesh.apiserver.service.type=LoadBalancer
+      --set clustermesh.apiserver.service.annotations."cloud\.google\.com/load-balancer-type"=Internal
+
+    It is also possible to use an IP address from a pre-defined subnet. In this
+    example, ``gke-vip-subnet`` is the name of the subnet that must
+    have been created before ``helm install cilium`` (see `Google
+    documentation for details
+    <https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#lb_subnet>`_):
+    
+    .. parsed-literal::
+
+      --set clustermesh.apiserver.service.annotations."networking\.gke\.io/internal-load-balancer-subnet"="gke-vip-subnet"
+      --set clustermesh.apiserver.service.loadBalancerIP="xxx.xxx.xxx.xxx"
+
+    ``xxx.xxx.xxx.xxx`` must be an IP from ``gke-vip-subnet``.
+
+    If not using a pre-set IP, get the service IP with:
+
+    .. code:: bash
+
+      kubectl -n kube-system get svc clustermesh-apiserver -o jsonpath='{.status.loadBalancer.ingress[0].ip}{"\n"}'
+
+  .. group-tab:: AWS
+
+    .. parsed-literal::
+
+      --set clustermesh.apiserver.service.type=LoadBalancer
+      --set clustermesh.apiserver.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-internal"="true"
+
+    If not using a pre-set IP, get the service IP with:
+
+    .. code:: bash
+
+      kubectl -n kube-system get svc clustermesh-apiserver -o jsonpath='{.status.loadBalancer.ingress[0].ip}{"\n"}'
+
+.. note::
+
+   Make sure that you use the namespace in which cilium is
+   running. Depending on which installation method you chose, this
+   could be ``kube-system`` or ``cilium``.
+
+Tell your cluster about external workloads
+==========================================
+
+To allow an external workload to join your cluster, the cluster must
+be informed about each such workload. This is done by adding a
+``CiliumExternalWorkload`` (CEW) CRD for each external workload. CEW
+CRD specifies the name, namespace, and labels for the workload. For
+now you must also allocate a small IP CIDR that must be unique to the
+workload. For example, for a VM named ``runtime`` that is to join the
+``default`` namespace, you could create a file ``runtime.yaml`` with
+the following contents:
+
+.. parsed-literal ::
+
+    apiVersion: cilium.io/v2
+    kind: CiliumExternalWorkload
+    metadata:
+      name: runtime
+      namespace: default
+      labels:
+        app: runtime
+    spec:
+      ipv4-alloc-cidr: 10.192.1.0/30
```

Comment:
It'd be nice to include something like this under `examples/`, use an `include` statement to include it, and ideally use the same example in the CI so we can validate that the example works.

(Reasonable as a followup)